### PR TITLE
Fixed typo

### PR DESCRIPTION
**Pull request by leeford**
Fixed typo - changed 'light' to 'lite'

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/direct-routing-plan-media-bypass.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ But let's assume that a user is in the same building or network as the SBC. For
 
 ![Shows signaling and media flow with media bypass](media/direct-routing-media-bypass-2.png)
 
-Media bypass leverages protocols called Interactive Connectivity Establishment (ICE) on the Teams client and ICE light on the SBC. These protocols enable Direct Routing to use the most direct media path for optimal quality. ICE and ICE Lite are WebRTC standards. For detailed information about these protocols, see RFC 5245.
+Media bypass leverages protocols called Interactive Connectivity Establishment (ICE) on the Teams client and ICE lite on the SBC. These protocols enable Direct Routing to use the most direct media path for optimal quality. ICE and ICE Lite are WebRTC standards. For detailed information about these protocols, see RFC 5245.
```
Comment (nenonix) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
Media bypass leverage protocols called Interactive Connectivity Establishment (ICE) enable Direct Routing on the Teams client and ICE lite on the SBC. These protocols use the most direct media path for optimal quality. ICE and ICE Lite are WebRTC standards. For detailed information about these protocols, see RFC 5245.
```

**Comment by scanum** [comment]
@CarolynRowe  changes look correct. Could you please confirm and merge? Thanks!

**Comment by scanum** [comment]
@CarolynRowe Changes look correct. Could you please review and merge? Thanks!

**Comment by CarolynRowe** [comment]
Let's discuss these changes.  I'm OK with the first change, but have questions about the other changes.

**Pull request merged**